spec/api/coffee_shops_endpoint_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rack/test'
+
+RSpec.describe Api::V1::CoffeeShopsRoute do
+  include Rack::Test::Methods
+
+  def app
+    Api::V1::CoffeeShopsRoute
+  end
+
+  let(:csv_url) { 'https://raw.githubusercontent.com/Agilefreaks/test_oop/master/coffee_shops.csv' }
```

Comment:
Using the remote csv is usually not a good practice, makes your tests brittle, any other approach that we can have here?

Reply:
We can stub the request like we do in other tests, but i wanted for this one to be more realistic. Maybe we can use something like [vcr](https://github.com/vcr/vcr), so we do not have to do a real request every time.